examples/customer-api/server.ts

```diff
@@ -59,6 +59,7 @@ export default {
         session,
         customerAccountId: env.PUBLIC_CUSTOMER_ACCOUNT_ID,
         customerAccountUrl: env.PUBLIC_CUSTOMER_ACCOUNT_URL,
+        customerApiVersion: '2024-01',
```

Comment:
Similar to storefront API, I feel like we should have the default API version as a constant in hydrogen package and display warming if the API version user pass in is differ than our target